src/middlewares/json.js

```diff
@@ -39,4 +44,8 @@ module.exports = class extends Middleware {
 		return stream;
 	}
 
+	hasBody(request) {
+		return request.headers['transfer-encoding'] !== undefined || !isNaN(request.headers['content-length']);
```

Comment:
Uh... https://stackoverflow.com/questions/32710847/what-is-the-best-way-to-check-for-empty-request-body

Reply:
Is `transfer-encoding` even required?

Reply:
hasBody isnt checking if the body is empty. its checking if the request has such headers that would be sent on a request with a body.
[Transfer-Encoding Documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding)